installer/helm/chart/volcano/templates/admission.yaml

```diff
@@ -133,6 +133,7 @@ spec:
             - --admission-conf=/admission.local.config/configmap/{{base .Values.basic.admission_config_file}}
             - --webhook-namespace={{ .Release.Namespace }}
             - --webhook-service-name={{ .Release.Name }}-admission-service
+            - --scheduler-name={{ .Values.basic.scheduler_name }}
```

Comment:
There is no need to add this parameter by default

Reply:
Yeah we just need to add it to `webhook.yaml`.

Reply:
> There is no need to add this parameter by default

I am confused, why in the `admission.yaml` file this parameter by default, if we change `scheduler.yaml` and `controllers.yaml` two deployment for scheduler-name , than use create pod can skip webhook step? 

Shouldn't all services use the same scheduler-name?